packages/manager/modules/log-to-customer/src/live-tail/template.html

```diff
@@ -31,63 +35,160 @@
     </description>
 
     <tile>
```

Comment:
**niptick** : log tiles could be defined in a standalone component.

Live tail template : 
```
<log-live-tail>
    <description>...</description>
    <tile><log-tiles ... /></tile>
</log-live-tail>
```